src/resolve/resolved-dns-query.h

```diff
@@ -64,7 +64,7 @@ struct DnsQuery {
         int ifindex;
 
         DnsTransactionState state;
-        unsigned n_cname_redirects;
+        uint8_t n_cname_redirects;
```

Comment:
similar. this is between a 32bit enum, and a native width ptr. so using 32bit shuld be fine...